lte/gateway/c/core/oai/tasks/s1ap/s1ap_mme_nas_procedures.c

```diff
@@ -76,6 +76,8 @@
 #include "lte/gateway/c/core/oai/tasks/s1ap/s1ap_common.h"
 #include "orc8r/gateway/c/common/service303/includes/MetricsHelpers.h"
 
+#define EXT_UE_AMBR_UL 1000000000
+#define EXT_UE_AMBR_DL 1000000000
```

Comment:
should not these be 10Gbps? I am counting 9 zeros here.

Reply:
Updated